checkov/common/output/cyclonedx.py

```diff
@@ -188,6 +188,10 @@ def create_library_component(self, resource: Record | ExtraResource, check_type:
           <name>flask</name>
           <version>0.6</version>
           <purl>pkg:pypi/cli_repo/pd/requirements.txt/flask@0.6</purl>
+          <properties>
+            <property name="startLine">5</property>
```

Comment:
What are these numbers? 5 and 6?

Reply:
5 is the start line number, 6 is the end line number (this is just an example)